src/lib/SpotlightTour.provider.tsx

```diff
@@ -3,40 +3,88 @@ import { ColorValue, LayoutRectangle } from "react-native";
 
 import { ChildFn, isChildFunction, isPromise } from "../helpers/common";
 
-import { SpotlightTour, SpotlightTourContext, SpotlightTourCtx, TourStep } from "./SpotlightTour.context";
+import { Position, SpotlightTour, SpotlightTourContext, SpotlightTourCtx, TourStep } from "./SpotlightTour.context";
 import { TourOverlay, TourOverlayRef } from "./tour-overlay/TourOverlay.component";
 
+export interface OSConfig<T> {
+  android: T;
+  ios: T;
+}
+
 interface SpotlightTourProviderProps {
+  /**
+   * The children to render in the provider. It accepts either a React
+   * component, or a function that returns a React component. When the child is
+   * a funtion, the `SpotlightTour` context can be accessed from the first
+   * argument.
+   */
   children: React.ReactNode | ChildFn<SpotlightTour>;
+  /**
+   * The color of the overlay of the tour.
+   *
+   * @default black
+   */
   overlayColor?: ColorValue;
-  overlayOpacity?: number | string;
+  /**
+   * The opacity applied to the overlay of the tour (between 0 to 1).
+   *
+   * @default 0.45
+   */
+  overlayOpacity?: number;
+  /**
+   * An array of `TourStep` objects that define each step of the tour.
+   */
   steps: TourStep[];
+  /**
+   * Define if the animations in the tour should use the native driver or not.
+   * A boolean can be used to apply the same value to both Android and iOS, or
+   * an object with `android` and `ios` keys can be used to define a value for
+   * each OS.
+   *
+   * @default false
+   */
+  nativeDriver?: boolean | OSConfig<boolean>;
 }
 
+const ZERO_SPOT: LayoutRectangle = {
+  height: 0,
+  width: 0,
+  x: 0,
+  y: 0
+};
+
 export const SpotlightTourProvider = React.forwardRef<SpotlightTour, SpotlightTourProviderProps>((props, ref) => {
-  const { children, overlayColor, overlayOpacity, steps } = props;
+  const {
+    children,
+    overlayColor = "black",
+    overlayOpacity = 0.45,
+    steps,
+    nativeDriver = false
```

Comment:
Why do we choose to disable `nativeDriver` by default?

Reply:
This used to be the default behavior because of all the issues we had running the animations in the native thread. But now that they have been solved, I think we can default it to true 🙂 thanks!